client/my-sites/post-relative-time-status/index.jsx

```diff
@@ -21,118 +20,164 @@ class PostRelativeTime extends React.PureComponent {
-		} else if ( status === 'future' ) {
-			const moment = this.props.moment;
-			const now = moment();
-			const scheduledDate = moment( this.props.post.date );
-			// If the content is scheduled to be release within a year, do not display the year at the end
-			const scheduledTime = scheduledDate.calendar( null, {
-				sameElse: this.props.translate( 'll [at] LT', {
-					comment:
-						'll refers to date (eg. 21 Apr) & LT refers to time (eg. 18:00) - "at" is translated',
-				} ),
-			} );
-
-			const displayScheduleTime =
-				scheduledDate.diff( now, 'years' ) > 0
-					? scheduledTime
-					: scheduledTime.replace( scheduledDate.format( 'Y' ), '' );
-
-			statusText = this.props.translate( 'scheduled for %(displayScheduleTime)s', {
-				comment: '%(displayScheduleTime)s is when a scheduled post is set to be published',
+		} else if ( status === 'trash' ) {
+			statusClassName += ' is-trash';
+			statusIcon = 'trash';
+			const displayScheduleTime = this.getDisplayedTimeForLabel();
+			statusText = this.props.translate( 'trashed %(displayScheduleTime)s', {
+				comment: '%(displayScheduleTime)s is when a post or page was trashed',
 				args: {
 					displayScheduleTime,
 				},
 			} );
+		} else if ( status === 'future' ) {
 			statusClassName += ' is-scheduled';
 			statusIcon = 'calendar';
-		} else if ( status === 'trash' ) {
-			statusText = this.props.translate( 'trashed' );
-			statusClassName += ' is-trash';
-			statusIcon = 'trash';
-		} else if ( this.props.includeBasicStatus ) {
-			if ( status === 'draft' ) {
-				statusText = this.props.translate( 'draft' );
-			} else if ( status === 'publish' ) {
-				statusText = this.props.translate( 'published' );
-			} else if ( status === 'new' ) {
-				statusText = this.props.translate( 'Publish immediately' );
-			}
+			const displayScheduleTime = this.getDisplayedTimeForLabel();
+			statusText = this.props.translate( 'scheduled %(displayScheduleTime)s', {
+				comment: '%(displayScheduleTime)s is when a scheduled post or page is set to be published',
+				args: {
+					displayScheduleTime,
+				},
+			} );
+		} else if ( status === 'draft' || status === 'pending' ) {
+			const displayScheduleTime = this.getDisplayedTimeForLabel();
+			statusText = this.props.translate( 'draft last modified %(displayScheduleTime)s', {
+				comment: '%(displayScheduleTime)s is when a draft post or page was last modified',
+				args: {
+					displayScheduleTime,
+				},
+			} );
+		} else if ( status === 'publish' ) {
+			const displayScheduleTime = this.getDisplayedTimeForLabel();
+			statusText = this.props.translate( 'published %(displayScheduleTime)s', {
+				comment: '%(displayScheduleTime)s is when a post or page was last modified',
+				args: {
+					displayScheduleTime,
+				},
+			} );
+		} else if ( status === 'private' ) {
+			const displayScheduleTime = this.getDisplayedTimeForLabel();
+			statusText = this.props.translate( 'private last modified %(displayScheduleTime)s', {
```

Comment:
Private posts should be considered as published as well. I noted we add a "Private" prefix to the title of these posts, so that should be enough: 

<img width="275" alt="Screen Shot 2020-04-27 at 11 55 26" src="https://user-images.githubusercontent.com/1233880/80359349-01762c00-887e-11ea-9942-5c37abeef556.png">

Ideally we could flag these posts using a label similar to the "Sticky"/"Pending reviews" ones and remove the title prefix, but it's fine to handle that in a follow-up.

Reply:
I did create an issue with that in mind a few days ago:
https://github.com/Automattic/wp-calypso/issues/41245

I'll reference it here to follow-up